doc/dev/quick_start.rst

```diff
@@ -101,101 +102,19 @@ PennyLane are also supported via the decomposition mechanism in Catalyst. For ex
         qml.adjoint(qml.SingleExcitation(jnp.pi / 3, wires=[0, 1]))
         return qml.state()
 
-In addition, you can jit most of :doc:`PennyLane templates <introduction/templates>` to easily construct and evaluate
-more complex quantum circuits; see below for the list of currently supported operations and templates.
+In addition, you can qjit most of :doc:`PennyLane templates <introduction/templates>` to easily construct and evaluate
+more complex quantum circuits.
 
 .. important::
 
    Decomposition will generally happen in accordance with the specification provided by devices,
-   which can vary from device to device (e.g. ``default.qubit`` and ``lightning.qubit`` might
+   which can vary from device to device (e.g., ``default.qubit`` and ``lightning.qubit`` might
    decompose quite differently.)
    However, Catalyst's decomposition logic will differ in the following cases:
 
    1. All :class:`qml.Controlled <pennylane.ops.op_math.Controlled>` operations will decompose to :class:`qml.QubitUnitary <pennylane.QubitUnitary>` operations.
    2. The set of operations supported by Catalyst itself can in some instances lead to additional decompositions compared to the device itself.
 
-.. raw:: html
-
-    <div class="summary-table">
-
-.. details::
-    :title: List of supported native operations
```

Comment:
I've removed these lists of supported native operations/observables from the quickstart as they no longer make sense; we now have multiple devices, and each device may support different operations.

Reply:
You're probably right for the most part, but I think on some level this was important to list the base capabilities in Catalyst, for example that only expval and var support observables.

Reply:
Yep! Yeah I'm not even sure that the op/obs list was even up to date in terms of 'operations natively supported through the catalyst machinery'.